moveit_core/smoothing_plugins/src/smoothing_base_class.cpp

```diff
@@ -0,0 +1 @@
+#include <moveit/smoothing_plugins/smoothing_base_class.h>
```

Comment:
Could you at least provide some basic structure around this, like header and description on what this is? I also don't see this being used anywhere, but I guess it's required for building a standalone library for the virtual base class.

Reply:
Good catch, this cpp file actually isn't needed at all. I just forgot to delete it when messing around earlier. Deleted.